app/views/groups/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if current_user %>
+  <div class="container-fluid d-flex flex-column align-items-center">
+    <% unless @groups.empty? %>
+      <% @groups.each do |group| %>
+        <div class="card mt-2 mb-1" style="width: 95%;">
+          <div class="card-body d-flex">
+            <div>
+              <img src=<%= group.icon %> class="category-icon me-2" alt="category icon">
+            </div>
+            <div class="w-75 d-flex justify-content-between align-items-center mx-2">
+              <div class="d-flex flex-column justify-content-center">
+                <%= link_to "#{group.name}", group_path(group.id), class:"card-title category-name fs-5 stretched-link" %>
+                <p class="card-text mt-2"><small class="text-muted"><%=group.created_at.strftime("%d %b %Y")%></small></p>
+              </div> 
+              <p class="card-text fw-semibold">&<%= group.entities.sum(:amount) %></p>             
+        </div>
+      <% end %>
+    <% else %>
+      <h3 class="mt-4">You don't have any categories yet.</h2>
+    
+    <% end %>
+    <%= link_to 'Add a new Category', new_group_path, class: "btn green btn-outline-info text-white" %>
```

Comment:
- [ ] As requested by the previous reviewer, the green button should stick to the bottom of the page.
![image](https://github.com/LoviMadaan/spendometer/assets/91486791/4d642c23-2186-46c4-94c3-70f0a4f04bc3)
- [ ] Also when I click the green button I get the following error: 
![image](https://github.com/LoviMadaan/spendometer/assets/91486791/770a1714-8cfa-4690-8d91-806654dec14a)

Reply:
Resolved

![Screenshot 2023-08-06 032448](https://github.com/LoviMadaan/spendometer/assets/119787828/fea80854-17c4-4b13-9211-4b5cc2b482bc)